server/src/test/java/org/candlepin/test/DatabaseTestFixture.java

```diff
@@ -157,6 +158,7 @@
     @Inject protected PermissionFactory permissionFactory;
 
     @Inject protected ResourceLocatorMap locatorMap;
+    @Inject protected MethodLocator methodLocator;
```

Comment:
The changes below are good, but mean that these should not be injected (since we create them ourselves during ```init```).

Reply:
Done.